src/component/utils/get-name-for-type/index.ts

```diff
@@ -0,0 +1,10 @@
+import { startCase } from 'lodash/fp';
+
+type Type = {
```

Comment:
actually you've got a def of Type here, maybe can use in the other file too if we put it in its own file and import around the place